src/main/java/org/zalando/nakadi/domain/Timeline.java

```diff
@@ -151,6 +151,14 @@ public boolean isFake() {
         return fake;
     }
 
+    public boolean isOpen() {
+        return getSwitchedAt() == null;
+    }
+
+    public boolean isClosed() {
```

Comment:
I don't know why, but I feel strange having both of these methods. I think the more traditional way is to have only one of them (so that users can do `!` themselves)
But that is just my opinion.